### PR TITLE
Extensions TSDoc edits

### PR DESCRIPTION
**Pull request by kevinthecheung**
Hey there! So you want to contribute to a Firebase SDK? 
Before you file this pull request, please read these guidelines:

### Discussion

  * Read the contribution guidelines (CONTRIBUTING.md).
  * If this has been discussed in an issue, make sure to link to the issue here. 
    If not, go file an issue about this **before creating a pull request** to discuss.

### Testing

  * Make sure all existing tests in the repository pass after your change.
  * If you fixed a bug or added a feature, add a new test to cover your code.

### API Changes

  * At this time we cannot accept changes that affect the public API.  If you'd like to help 
    us make Firebase APIs better, please propose your change in an issue so that we 
    can discuss it together.

### CODE REVIEW

File: src/extensions/extensions-api.ts [hunk 1]
```diff
@@ -16,14 +16,22 @@
  */
 
 /**
- * SettableProcessingState represents all the Processing states that can be set on an ExtensionInstance's runtimeData.
+ * SettableProcessingState represents all the Processing states that can be set
+ * on an ExtensionInstance's runtimeData.
```
Comment (ifielker) [inline src/extensions/extensions-api.ts:20]: ExtensionInstance should be 2 words?

**Review by ifielker — APPROVED** [review]

File: src/extensions/extensions-api.ts (hunk 1 above)
Comment (lahirumaramba) [inline src/extensions/extensions-api.ts:20]: Does `ExtensionInstance` need backticks? or should it be "Extension instance's"

File: src/extensions/extensions-api.ts [hunk 2]
```diff
@@ -16,14 +16,22 @@
  */
 
 /**
- * SettableProcessingState represents all the Processing states that can be set on an ExtensionInstance's runtimeData.
+ * SettableProcessingState represents all the Processing states that can be set
```
Comment (lahirumaramba) [inline src/extensions/extensions-api.ts:19]: Should we back-tick `Processing`?

**Review by ifielker — APPROVED** [review]

**Review by ifielker — APPROVED** [review]

File: src/extensions/extensions-api.ts [hunk 3]
```diff
@@ -16,14 +16,22 @@
  */
 
 /**
- * SettableProcessingState represents all the Processing states that can be set on an ExtensionInstance's runtimeData.
+ * SettableProcessingState represents all the processing states that can be set
```
Comment (lahirumaramba) [inline src/extensions/extensions-api.ts:19]: I think we can remove `SettableProcessingState` from here and just say `Represents all the Processing states...`
```suggestion
 * Represents all the processing states that can be set
```

File: src/extensions/extensions-api.ts [hunk 4]
```diff
@@ -16,14 +16,22 @@
...
+ * on an Extension instance's runtimeData.
```
Comment (lahirumaramba) [inline src/extensions/extensions-api.ts:20]: Should `runtimeData` back-ticked?

File: src/extensions/extensions-api.ts [hunk 5]
```diff
@@ -16,14 +16,22 @@
...
  * - PROCESSING_COMPLETE: Lifecycle event work completed with no errors.
+ * 
  * - PROCESSING_WARNING: Lifecycle event work succeeded partially,
  *                        or something happened that the user should be warned about.
+ * 
  * - PROCESSING_FAILED: Lifecycle event work failed completely,
  *                        but the instance will still work correctly going forward.
- * - If the extension instance is in a broken state due to the errors, instead set FatalError.
+ * 
+ * - If the extension instance is in a broken state due to errors, instead set FatalError.
```
Comment (lahirumaramba) [inline src/extensions/extensions-api.ts:33]:
```suggestion
 * - If the extension instance is in a broken state due to errors, instead set `FatalError`.
```

**Review by lahirumaramba — APPROVED** [review]

**Comment by lahirumaramba** [comment]
Thank you, @kevinthecheung ! LGTM!

**Review by ifielker — APPROVED** [review]

**Pull request merged**